### PR TITLE
Add more JVM monitoring

### PR DESCRIPTION
**Pull request by Willymontaz**
* Add a tag choice on JVM dashboard
 * Add missing header fields on forwarder (user, pid, java main class)
 * Turn on JVM statistics on NodeManagerModule
 * Add missing header field on nodemanager (java main class)

Replaces https://github.com/criteo/garmadon/pull/121

### CODE REVIEW

File: agent/src/main/java/com/criteo/hadoop/garmadon/agent/headers/RessourceManagerHeader.java [hunk 1]
```diff
@@ -11,9 +12,9 @@ private RessourceManagerHeader() {
 
     private Header createCachedHeader() {
         return Header.newBuilder()
-                .withHostname(Utils.getHostname())
-                .withUser(Utils.getUser())
-                .withPid(Utils.getPid())
+                .withHostname(HeaderUtils.getHostname())
+                .withUser(HeaderUtils.getUser())
+                .withPid(HeaderUtils.getPid())
```
Comment (ashangit) [inline agent/src/main/java/com/criteo/hadoop/garmadon/agent/headers/RessourceManagerHeader.java]: .withMainClass(HeaderUtils.getArrayJavaCommandLine()[0]) ?

File: agent/src/main/java/com/criteo/hadoop/garmadon/agent/modules/NodeManagerModule.java [hunk 2]
```diff
@@ -3,12 +3,16 @@
...
 import com.criteo.hadoop.garmadon.agent.headers.NodemanagerHeader;
 import com.criteo.hadoop.garmadon.agent.tracers.hadoop.nodemanager.ContainerResourceMonitoringTracer;
+import com.criteo.hadoop.garmadon.agent.tracers.jvm.JVMStatisticsTracer;
 
 import java.lang.instrument.Instrumentation;
 
 public class NodeManagerModule implements GarmadonAgentModule {
     @Override
     public void setup(Instrumentation instrumentation, AsyncEventProcessor eventProcessor) {
+        JVMStatisticsTracer.setup((timestamp, event) ->
```
Comment (ashangit) [inline agent/src/main/java/com/criteo/hadoop/garmadon/agent/modules/NodeManagerModule.java]: Should we also add this on ResourceManagerModule

File: agent/src/main/java/com/criteo/hadoop/garmadon/agent/headers/RessourceManagerHeader.java (hunk 1 above)
Comment (Willymontaz) [reply agent/src/main/java/com/criteo/hadoop/garmadon/agent/headers/RessourceManagerHeader.java]: was not necessary for the purpose of the dashboard but yes I will add it

File: agent/src/main/java/com/criteo/hadoop/garmadon/agent/modules/NodeManagerModule.java (hunk 2 above)
Comment (Willymontaz) [reply agent/src/main/java/com/criteo/hadoop/garmadon/agent/modules/NodeManagerModule.java]: it makes sense

**Pull request merged**